Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
   gem 'simplecov'
   gem 'factory_bot_rails'
   gem 'faker'
+  gem 'pry-rescue'
+  gem 'pry-stack_explorer'
```

Comment:
I added the `pre-rescue` gem when debugging and decided to leave it in because it seems like a helpful gem. If you run `bundle exec rescue rspec` it will automatically put you in a pry if there are any errors, which is really nice when you don't know exactly where/why they are occurring. It also works with `rescue rails s` if you want to debug errors on localhost.

Reply:
That is awesome!!! I am going to use this gem all the time now.

Reply:
this sounds awesome but also I don't 100% understand :tada: